docs/input.md

```diff
@@ -126,6 +128,8 @@ G15511    XX    0    C09DFN    pathToFiles/G15511.C09DFN.md.recal.bam    pathToF
 G15511    XX    1    D0ENMT    pathToFiles/G15511.D0ENMT.md.recal.bam    pathToFiles/G15511.D0ENMT.md.recal.bai
 ```
 
+When starting Sarek from the mapping or recalibrate steps, a TSV file is generated automatically after the recalibration processes. This TSV file is stored under `results/Preprocessing/TSV/recalibrated.tsv` and can be used to restart Sarek from the recalibrated BAM files, giving it as `--input` and setting the step `--step variantcalling`.
```

Comment:
Same here with `--step variantcalling`

Reply:
Same here: we could add that a TSV file is generated for each sample as well

Reply:
ok, great, I'll add this as well